CMakeLists.txt

```
@@ -16,6 +16,8 @@ option(IOTA_TESTS "Enable IOTA Client library test cases" ON)
 option(IOTA_ASAN_ENABLED "Enable Address sanitizer" OFF)
 option(ENABLED_JEMALLOC "Enable jemalloc memory allocator" OFF)
 option(IOTA_EXAMPLES "Build example application" OFF)
+option(IOTA_BENCHMARK "Build applications for a memory benchmarking" OFF)
+option(ENABLE_MTRACE "Enable mtrace in applications for memory benchmarking" OFF)
```

only benchmark applications have this feature. 

```suggestion
option(ENABLE_MTRACE "Enable mtrace in benchmark applications" OFF)
```

---

Done. Thank you.